src/release/breaking-changes/android-14-nonlinear-text-scaling-migration.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Android 14 nonlinear font scaling enabled after v3.14
+description: >
+  New Android 14 nonlinear font scaling feature is enabled in Flutter after v3.14.
+---
+
+## Summary
+
+Android 14 introduced nonlinear font scaling to 200%. It may change how your app 
+looks when the user changes the accessiblity text scaling in system preferences.
+
+## Background
+
+The [Android 14 nonlinear font scaling][] feature prevents excessive accessibility 
+font scaling by scaling larger text at a lesser rate when the user increases the 
+text scaling value in system preferences.
+
+## Migration guide
+
+As suggested by the 
+[Android 14 feature overview][Android 14 nonlinear font scaling], perform UI 
+testing with the maximum font size enabled (200%) to ensure that your app 
+applies the font sizes correctly and can accommodate larger font sizes without
+impacting usability.
```

Comment:
**suggestion:** Use [semantic breaks](https://sembr.org/). Write in active voice per [GSG](https://developers.google.com/style/voice).
```suggestion
As the [Android 14 feature overview][Android 14 nonlinear font scaling] suggests,
test your UI with the maximum font size enabled (200%).
This should verify that your app can apply the font sizes correctly
and can accommodate larger font sizes without impacting usability.
```